espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Router.kt

```diff
@@ -139,6 +139,7 @@ internal class Router {
         routeMap.addRoute(RouteDefinition(Method.POST, "/session/:sessionId/appium/execute_mobile/register_idling_resources", RegisterIdlingResources(), IdlingResourcesParams::class.java))
         routeMap.addRoute(RouteDefinition(Method.POST, "/session/:sessionId/appium/execute_mobile/unregister_idling_resources", UnregisterIdlingResources(), IdlingResourcesParams::class.java))
         routeMap.addRoute(RouteDefinition(Method.GET, "/session/:sessionId/appium/execute_mobile/list_idling_resources", ListIdlingResources(), AppiumParams::class.java))
+        routeMap.addRoute(RouteDefinition(Method.POST, "/session/:sessionId/appium/execute_mobile/select_driver", SelectDriver(), DriverParams::class.java))
```

Comment:
there should also be a method to get the currently selected driver. Also, it would probably make sense to move this functionality into Settings API

Reply:
See 
https://github.com/appium/appium-mac2-driver/pull/28/
https://github.com/appium/appium-mac2-driver/pull/30
as an example on how to add settings API

Reply:
added